lib/redis_client_options.rb

```diff
@@ -14,6 +14,7 @@ def default_redis_options
     opts[:password] = config[:password] if config[:password]
     opts[:timeout]  = config[:timeout]  if config[:timeout]
     opts[:db]       = config[:database] if config[:database]
+    opts[:scheme]   = config[:transport] if config[:transport]
```

Comment:
looks like this needs to be changed to `opts[:transport]`

Reply:
correct me if I am wrong but are these not the options passed to the redis client, in which `scheme` is the correct option we are trying to set?

Reply:
ah I had not looked at this code for a while and misunderstood based on the diff github showed me.